cucumber/authenticators/features/authn_oidc.feature

```diff
@@ -81,8 +81,13 @@ Feature: Users can authneticate with OIDC authenticator
   Scenario: Non-existing username in ID token is denied
     Given I get authorization code for username "not_in_conjur" and password "not_in_conjur"
     And I fetch an ID Token
+    And I save the log data from bookmark
     When I authenticate via OIDC with id token
     Then it is unauthorized
+    And The log filtered from bookmark contains messages:
+    """
+    Authentication Error: #<Authentication::NotDefinedInConjur: User 'not_in_conjur' is not defined in Conjur
```

Comment:
I would match only on `Authentication::NotDefinedInConjur` and, if you want to be extra safe, `not_in_conjur`.  This way the test will be robust against (relatively likely) changes in the particular wording of the error message.